code/game/machinery/rechargestation.dm

```diff
@@ -18,6 +19,16 @@
 	var/capacitor_max = 0 //combined max power the capacitors can hold
 	machine_flags = SCREWTOGGLE | CROWDESTROY | WRENCHMOVE | EJECTNOTDEL
 
+	hack_abilities = list(
+		/datum/malfhack_ability/oneuse/make_autoborger,
```

Comment:
I'm also concerned about the ability to turn EVERY CYBORG CHARGER into an autoborger. Just one autoborger is considered an extremely valuable investment that shuts off your shunt ability right now, and you have one to defend in a safe place. This lets you have stealthy autoborgers all over the station, in maint and near major hallways.

I'm also concerned because this is an undocumented change in a 305 files changed PR. You *need* to be transparent with people in major reworks because they can't be expected to read all of it. I'm disappointed.

Reply:
I raised the price to 100 power, I think I will limit it to a maximum of one autoborger.